index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <title>Personal Portfolio</title>
+
```

Comment:
Nice job so far 👍 but it would be nice you delete these empty lines ♻️ . 
They make your codebase tardy

Reply:
- [ ] Good job so far for using the inline comments :+1:  But please phrase the comments politely, as suggestions instead of orders for example instead of "It would be ..." you could have said, "I think  ..."  :smile: